mappings/net/minecraft/world/dimension/YLevels.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_6101 net/minecraft/world/dimension/YLevels
+	FIELD field_31428 OVERWORLD_MIN_Y I
+	FIELD field_31429 OVERWORLD_HEIGHT I
+	FIELD field_31430 OVERWORLD_GENERATION_HEIGHT I
+	FIELD field_31431 OVERWORLD_LOGICAL_HEIGHT I
+	FIELD field_31432 NETHER_MIN_Y I
+	FIELD field_31433 NETHER_HEIGHT I
+	FIELD field_31434 NETHER_GENERATION_HEIGHT I
+	FIELD field_31435 NETHER_LOGICAL_HEIGHT I
+	FIELD field_31436 END_MIN_Y I
+	FIELD field_31437 END_HEIGHT I
+	FIELD field_31438 END_GENERATION_HEIGHT I
+	FIELD field_31439 END_LOGICAL_HEIGHT I
```

Comment:
Yeah, might need to remember to unpick these

Reply:
The problem with unpicking them is that a lot of them are the same number - for example, both `NETHER_MIN_Y` and `END_MIN_Y` are 0, and both `OVERWORLD_HEIGHT` and `OVERWORLD_LOGICAL_HEIGHT` are 386.
Unpick wouldn't know which one to use.

Reply:
Hmm, then how do you know which field is which :thinking:

Reply:
> 
> 
> Hmm, then how do you know which field is which 🤔

I mean you can tell they're all grouped together by dimension in the order of Min Y, Height, Generation Height, Logical Height
(though to be fair each pair of generation heights and logical heights are identical, so those may technically be the wrong way around...)

Reply:
> (though to be fair each pair of generation heights and logical heights are identical, so those may technically be the wrong way around...)

I think we've rejected sollace's pr once before that map instance final constant fields based on their presumed order, and I wonder if our policy have hopefully changed